lienflash/lienflash-fe
# Tm/error handling

#### What’s this PR do?
- Update tests to reflect changes to implementation code
- Improve test coverage

#### Where should the reviewer start?
- All the files

#### How should this be manually tested?
- In terminal, run npm test -- --coverage --watchAll=false to see the passing tests and how covered everything is

#### Need Help/Support
- can't figure out how to test back button in job details - wondering if we test in app w/integration testing if it'll count towards the coverage?
